src/backends/torch/torchlib.cc

```diff
@@ -725,12 +773,32 @@ namespace dd
               }
             else if (_timeserie)
               {
-                if (_loss.empty() || _loss == "L1" || _loss == "l1")
-                  loss = torch::l1_loss(y_pred, y);
-                else if (_loss == "L2" || _loss == "l2" || _loss == "eucl")
-                  loss = torch::mse_loss(y_pred, y);
+                if (_template.find("nbeats") != std::string::npos)
```

Comment:
Can we wrap this up somewhere dedicated to the `nbeats` template ? Like `nbeat_native_template_loss_selection()` that would follow some generic signature ? This would allows simplifying the adding of more native templates.

Reply:
I am doing this directly in nbeats class, ie there will be here _module._native->loss() here, what do you think ?